src/Settings/GlobalSettings/SettingsGlobal.php

```diff
@@ -138,10 +123,12 @@ private function getAllSettingsSidebars(): array
 	 */
 	private function getAllSettings(): array
 	{
-		$allSettings = self::GLOBAL_SETTINGS;
+		$allSettings = [];
 
-		foreach (Integrations::ALL_INTEGRATIONS as $key => $integration) {
-			$allSettings[$key] = $integration['global'] ?? '';
+		foreach (Filters::ALL as $key => $integration) {
+			if (isset($integration['global'])) {
+				$allSettings[$key] = $integration['global'] ?? '';
+			}
```

Comment:
Shouldn't `$integration['global'] ?? ''` be enough (unnecessary check for isset)?

Reply:
not sure but it looks to me that it will pass the empty string to the array this way. 
I can refactor this for a return early method if you would like

Reply:
Sure, but my point is that the value of `$allSettings[$key]` will either be `$integration['global']` or an empty string,  if the `$integration['global']` is not empty and set. So the check above isn't necessary.

Or if you want the conditional to be necessary, then the `?? ''` part is not necessary.